tasks/main.yml

```diff
@@ -7,6 +10,10 @@
        dest={{ otp_bin_dir }}
        version={{ otp_version }}
 
+- name: Set OpenTripPlanner Session Timeout
+  lineinfile: dest={{ otp_bin_dir }}/src/client/WEB-INF/web.xml regexp="(^|\W)<session-timeout>30</session-timeout>$" line="\t\t<session-timeout>{{ otp_session_timeout_s }}</session-timeout>"
```

Comment:
Consider putting `regexp` and `line` on separate lines from `dest` so that easier to read without having to scroll horizontally:

```yaml
- name: Set OpenTripPlanner Session Timeout
  lineinfile: dest={{ otp_bin_dir }}/src/client/WEB-INF/web.xml
              regexp="(^|\W)<session-timeout>30</session-timeout>$"
              line="\t\t<session-timeout>{{ otp_session_timeout_s }}</session-timeout>"
```